git.go

```diff
@@ -9,4 +9,4 @@ import (
 func GetBranchName() string {
 	output, _ := exec.Command("git", "rev-parse", "--abbrev-ref", "HEAD").Output()
 	return strings.TrimSpace(string(output))
-}
+}
```

Comment:
Any reason to remove this newline? There might be some good reasons to keep it - https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
I'm just gonna switch to VS Code- JetBrains's linter just disagrees.

Reply:
Strange! `golint` would be the definitive linter here with regard to linters for go, so whatever `golint` says, that's the right thing. VS Code uses `golint` :) I use VS Code personally for go, though it tends to have some problems with variable renaming and other things, I imagine I just haven't spent enough time with the setup.